sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -175,6 +175,30 @@ namespace Azure { namespace Core { namespace Http {
     NetworkAuthenticationRequired = 511, ///< HTTP 511 Network Authentication Required.
   };
 
+  /**
+   * @brief Defines a range of bytes within an HTTP resource, starting at an `Offset` and ending at
+   * `Offset + Length - 1` inclusively.
+   *
+   */
+  struct Range
```

Comment:
I understand this is where we landed in the issue, but re-consider naming is `HttpRange`.

Reply:
It's already in `Http` namespace.